pkg/operator/ceph/object/spec.go

```diff
@@ -144,7 +144,9 @@ func (c *clusterConfig) makeDaemonContainer(rgwConfig *rgwConfig) v1.Container {
 					Port: intstr.FromInt(int(c.store.Spec.Gateway.Port)),
 				},
 			},
-			InitialDelaySeconds: 10,
+			InitialDelaySeconds: 30,
+			TimeoutSeconds:      20,
+			PeriodSeconds:       60,
```

Comment:
The interval is pretty high. What if the RGW becomes unhealthy in that time span? Then it wouldn't be removed fast enough.

Reply:
Yes, that's a possibility... but I have no solution for this, other than spanning more rgw as the load goes up which we can't do at the moment.

Reply:
Ok some updates, it seems that `TimeoutSeconds` default is 1 and not 10 (just like the code comment says). So I'm lowering this down a bit.

Reply:
Rahh nevermind, it's just me misreading...